scripts/auto_release/requirement.txt

```diff
@@ -1,6 +1,7 @@
 azure-identity
 python-dotenv==0.15.0
 ghapi==0.1.19
+PyGithub==1.55
```

Comment:
nit: rest of the repo doesn't depend on `pygithub`, we are on minversion 1.59.0 where we do though.